src/js/index.js

```diff
@@ -67,6 +65,10 @@ function setupShowcases() {
   const showcase = $('.showcases__project')
   const code = document.getElementById('showcase-code')
   const title = document.getElementById('showcase-title')
+  if (!code) {
```

Comment:
Nope. !showcase will always be false. OTOH, why jQuery for one of the selectors?